routing.rst

```diff
@@ -995,6 +995,136 @@ convert them automatically to their scalar values.
         }
     }
 
+Mapping Parameters
+~~~~~~~~~~~~~~~~~~
+
+By default, the name of the variable part (``{slug}`` for example) is the
+argument injected name to the method (``$slug``).
+
+You can change this behavior and define a mapping between variable part and
+an argument name with ``{variable_part_name:argument_name}``:
```

Comment:
I know it's very long, but maybe we can use this to be super explicit here:

`{route_parameter_name:controller_argument_name}`